src/components/Editor/GeneralInformation/BaseFields.vue

```diff
@@ -431,6 +431,33 @@
       </v-expand-transition>
     </v-col>
 
+    <!-- Upload logo -->
+    <v-col cols="12">
+      Logo:
+      <upload-files
+        :credential-info="{id:getField('id'),token:user().credentials.token}"
+        :initial-images="getField('urlForLogo')"
+        :upload-service-name="'uploadOneFilePerRequest'"
+        :base-api-endpoint="getAPIEndPoint()"
+        :allowed-file-size-mb="3"
+        mime-type="image/jpeg,image/gif,image/png"
+        file-key-name="logo"
+        title="logo"
+      />
+      <v-divider />
+      <upload-files
```

Comment:
The presence of this multiple-file-upload box means that formValid isn't true unless it's filled in. Since it should never be, as only a single logo is to be uploaded, then this second upload field should be removed.

Reply:
indeed. that was added for review purposes. it l will be removed.